src/ui/containers/DevContainer.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, useEffect } from 'react';
```

Comment:
this is maybe pretty odd and I can remove it if needed..

Basically a place to test queries in dev..